exe/Wrapper.hs

```diff
@@ -65,7 +66,7 @@ main = do
   logm $ "args:" ++ show args
 
   -- Get the cabal directory from the cradle
-  cradle <- findLocalCradle (d </> "File.hs")
+  cradle <- loadImplicitHieCradle (d </> "File.hs")
```

Comment:
```suggestion
  hieYaml <- findCradle (d </> "File.hs")              
  cradle <- maybe (loadImplicitHieCradle $ addTrailingPathSeparator d) loadCradle hieYaml 
```

something like that, otherwise we are not using any existing `hie.yaml` definitions, right?

Reply:
I'll take your word for it I am not familiar with the wrapper code, since I don't use wrapper.

Reply:
@fendor Are we still planing on using cabal-helper in wrapper or should I just tear it out.
What is it doing for us in wrapper?

Reply:
It is not used anymore, afaict. You already teared it out :)
It is doing the same thing as before, we are trying to find a cradle and based on that cradle we try to find the ghc version. If we find an ghc version, try to launch the correct hls version.